src/locale/i18n.ts

```diff
@@ -1,8 +1,10 @@
 // Don't remove -force from these because detection is VERY slow on low-end Android.
 // https://github.com/formatjs/formatjs/issues/4463#issuecomment-2176070577
 import '@formatjs/intl-locale/polyfill-force'
+import '@formatjs/intl-datetimeformat/polyfill-force'
```

Comment:
Just flagging we need to measure the startup time impact of this on a real android device. We've had pretty bad regressions in the past due to related changes.

Another thing we might want to consider is forking this block between iOS and Android. If this is really android-only, it would be good to avoid activating this on iOS.

Reply:
Actually maybe you can move this import to `time.android.ts`? Since that's the only place that needs that API. I realize this makes it a bit inconsistent but it does avoid potentially regressing something on iOS.

Reply:
I guess I'm not sure whether that would break the dynamic imports below. Do they rely on ordering?